server.js

```diff
@@ -24,12 +33,23 @@ router.get('/', function(req, res) {
     res.json({ message: 'Welcome to the More2Life App API!' });
 });
 
-// more routes for our API will happen here
 
 // REGISTER OUR ROUTES -------------------------------
 // all of our routes will be prefixed with /api
 app.use('/api', router);
 
+router.get('/feed', function (req, res) {
```

Comment:
Had a brief chat with Porter, but since this is really a straight-up collection of all of the feedItems we have, and our type/schema is feedItems, let's call it /feedItems